packages/api/src/post-build.js

```diff
@@ -192,6 +236,30 @@ class PostBuildHandler {
         handleError(res, err);
       }
     });
+
+    this.app.post('/api/build-gh', async (req, res, next) => {
```

Comment:
Is there another name for this than `gh`? There might be test runners other than GitHub that send data this way.

Reply:
renamed to v1. Then, every time a change is made the process can be adding new endpoint with higher version, updating action. And then after a year or so we can deprecate the old endpoint and remove that version to post to.